includes/Checker/CLI_Runner.php

```diff
@@ -178,6 +178,31 @@ protected function get_slug_param() {
 		return $slug;
 	}
 
+	/**
+	 * Initializes the runtime environment so that runtime checks can be run against a separate set of database tables.
+	 *
+	 * @since n.e.x.t
```

Comment:
```suggestion
	 * @since 1.3.0
```

Reply:
Good point! Updated in https://github.com/WordPress/plugin-check/pull/749/commits/3dcd99c78e95636d79e07f6e59c991977c22e8f9